collect_app/src/androidTest/java/org/odk/collect/android/feature/formmanagement/InvalidFormTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.odk.collect.android.feature.formmanagement
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain.chain
+
+@RunWith(AndroidJUnit4::class)
+class InvalidFormTest {
+    private var rule = CollectTestRule()
+
+    @get:Rule
+    var copyFormChain: RuleChain = chain().around(rule)
+
+    @Test
+    fun brokenForm_shouldNotBeVisibleOnFormList() {
+        rule.startAtMainMenu()
+            .copyForm("invalid-form.xml")
+            .clickFillBlankForm()
+            .checkIsSnackbarErrorVisible()
```

Comment:
This could probably be more precise. Let's assert on the actual text we expect here.

Reply:
Done.